src/components/BorrowerProfile/LendCta.vue

```diff
@@ -47,9 +47,16 @@
 					<p class="tw-text-h3 tw-pt-3 lg:tw-mb-3 tw-hidden lg:tw-inline-block">
 						{{ lgScreenheadline }}
 					</p>
-					<span class="tw-flex tw-pb-1 lg:tw-pb-2.5">
+					<span class="tw-flex tw-flex-wrap tw-pb-1 lg:tw-pb-2">
+						<!-- Highlighted matching text mobile  -->
+						<!-- padding-left class makes room for the sparkle icon
+						and makes sure long match text isnt covered -->
+						<span
+							class="tw-line-clamp-1 match-text tw-mb-0.5 md:tw-hidden tw-pl-3"
+							v-if="matchingTextVisibility && matchingHighlightExpShown"
```

Comment:
This could just be `matchingHighlightExpShown` because of check in `initializeMatchingHighlightExp`